ports/soh2/Ship of Harkinian 2.sh

```diff
@@ -15,6 +15,8 @@ fi
 source $controlfolder/control.txt
 get_controls
 
+# Source Device Info
+source $controlfolder/device_info.txt
 [ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
 
```

Comment:
Applying the new launch script template 

```suggestion
[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
get_controls
```